### ISSUE TITLE
PL/* language packages are not installed

### ISSUE THREAD

**Issue by ivan-kolmychek** [docker-library/postgres#114]
The `http://apt.postgresql.org` provides packages like `postgresql-plperl`, `postgresql-plpython`, `postgresql-plpython3` and `postgresql-pltcl` for them. 

I think, they should be installed alongside with `postgresql-contrib`, otherwise you'll get an error like 

```
PG::UndefinedFile: ERROR:  could not open extension control file "/usr/share/postgresql/
9.3/extension/plperl.control": No such file or directory
:       CREATE EXTENSION IF NOT EXISTS plperl
```

when you'll encounter the app that uses the one of `PL/*` languages for whatever reason.

I'm having this issue with `PL/Perl` and Dockerfile for `9.3` but there are also packages for `9.4` and `9.5`.

**Comment by cdaringe** [comment]
im struggling with this as well!  did you come to any conclusion?

alpine doesn't ship a `plpython` for postgres `9.5.x`/`9.5.10`, and im struggling to find documentation on how to load it.

i even downloaded the 9.5 source and compiled it, but still not clear which artifacts are relevant and where they go!

**Comment by cdaringe** [comment]
whoops, i commented in the wrong issue.  this issue is essentially closed (rejected) w/ the closure of #115

**Comment by qedrakmar** [comment]
I'm closing this, as reasoned by the above comments.
